src/pybind/mgr/dashboard/services/auth.py

```diff
@@ -43,7 +45,7 @@ def gen_token(cls, username):
             cls.init()
         ttl = mgr.get_module_option('jwt_token_ttl', cls.JWT_TOKEN_TTL)
         ttl = int(ttl)
-        now = int(time.mktime(time.gmtime()))
+        now = int(calendar.timegm(time.gmtime()))
```

Comment:
What about simply using 
```suggestion
        now = int(time.time())
```

```
time.time()
Return the time in seconds since the epoch as a floating point number. Note that even though the time is always returned as a floating point number, not all systems provide time with a better precision than 1 second.
```
https://docs.python.org/2/library/time.html